spec/system/pages/home_spec.rb

```diff
@@ -20,6 +20,26 @@
     expect(page).to_not have_link "Pipe Fitter", href: occupation_standard_path(pipe_fitter)
   end
 
+  it "displays the correct page title when page_title is present" do
+    allow(State).to receive(:find_by).and_return(build_stubbed(:state))
+    create(:occupation_standard, :with_work_processes, :with_data_import, title: "Mechanic")
+
+    visit home_page_path
```

Comment:
We can simplify this test if we go directly to `occupation_standards_path` and check that the title is correct. We can also remove the need to stub an State and create an occupation standard